content/api/trafiklab-apis/netex-regional/siri.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Siri
+aliases:
+- "api/netex-regional-siri"
+weight: 20 
+date: 2022-04-14
+---
+
+Siri is service Interface for Real Time Information.
+
+The Norwegian profile includes Estimated Timetables (ET), Situation Exchange (SX) and Vehicle Monitoring (VM) data.
+These are consistent with GTFS-RT TripUpdates, ServiceAlerts and VehiclePositions.
```

Comment:
More information about the profile version is needed here

Samtrafikens SIRI feed is based on version 1.1 of the Norwegian SIRI profile. This profile is based on SIRI 2.0. The full profile documentation can be found here: https://enturas.atlassian.net/wiki/spaces/PUBLIC/pages/637370420/Norwegian+SIRI+profile